rocketmq-flink/src/main/java/org/apache/rocketmq/flink/RocketMQSink.java

```diff
@@ -152,21 +166,36 @@ public void onException(Throwable throwable) {
     }
 
     private Message prepareMessage(IN input) {
-        String topic = topicSelector.getTopic(input);
-        String tag = (tag = topicSelector.getTag(input)) != null ? tag : "";
+        Validate.notNull(input, "the input is null");
```

Comment:
prepareMessage will throw runtime exception if you use validate mode in here. Is it what you expect?